accepted/2021/utf8/requirements/README.md

```diff
@@ -0,0 +1,26 @@
+# Requirements
+
+**DRAFT**
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+## [UTF-8 Strings should be pre-validated](validation.md)
+
+- allow us to do better optimizations, for example when enumerating runes
```

Comment:
As we discussed offline, I do not think this is the right choice for .NET. I think we should be more like golang and the validity of the encoding should be a soft requirement.

Reply:
Krzysztof, would it make sense to call these opinionated behaviors / areas of investigation rather than requirements? That more accurately describes that these are open for discussion and modification.

Reply:
> Krzysztof, would it make sense to call these opinionated behaviors / areas of investigation rather than requirements?

I want to get the skeleton/stub merged in so that we can then iterate on these specific docs individually and have focused discussions on them. So this approach sounds good to me.

Reply:
I agree, I will change this whole section to `Areas of investigation`